src/InteractionModNborhood.H

```diff
@@ -129,7 +138,7 @@ void InteractionModNborhood<AC,ACT,ACTD,A>::interactAgents( AC& a_agents, /*!< A
         {
             auto pair_ind = std::make_pair(mfi.index(), mfi.LocalTileIndex());
             auto bins_ptr = a_agents.getBins(pair_ind, ExaEpi::InteractionNames::nborhood);
-
+            auto m_at_work_ptr = a_agents.isAtWork(); /*! Flag to indicate if agents are at work */
```

Comment:
Change this to `at_work` or `flag_at_work`. `m_` in the beginning denotes member variable; this is a local variable. `_ptr` is not needed because this is a `bool`-type, not a pointer.